admin/metrics/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class OSFStatistic(models.Model):
+    users = models.IntegerField(verbose_name='OSF users', default=0)
+    delta_users = models.IntegerField(default=0)
+    unregistered_users = models.IntegerField(verbose_name='Unregistered users',
+                                             default=0)
+    projects = models.IntegerField(verbose_name='Number of projects',
+                                   default=0)
+    delta_projects = models.IntegerField(default=0)
+    public_projects = models.IntegerField(
+        verbose_name='Number of public projects', default=0)
+    delta_public_projects = models.IntegerField(default=0)
+    registered_projects = models.IntegerField(
+        verbose_name='Number of projects registered', default=0
+    )
+    delta_registered_projects = models.IntegerField(default=0)
+    date = models.DateTimeField(default=None)
```

Comment:
@whit537 Actually this is a really nice feature that comes with Django. If you look closely at the `scripts` directory you will see handwritten masterpieces of what Django can auto-generate (in most cases) for its own framework. Basically when you run `makemigrations` it generates the migration code for updating the database. In a more complex project there could be a series of these migration files and each instance of the server will know what the last migration it did. In this case since it is the first one, and even though I generated previous migrations I decided to make the commit a bit cleaner by squashing them into a single file. This way only `inv admin.manage migrate` needs to be run in order for a server to be updated.

So yes, it may look as though it is a duplication of code, but in reality it saves duplication of effort, unlike the parent Flask app. If you would like to read more on migrations please refer to https://docs.djangoproject.com/en/1.9/topics/migrations/